code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -37,20 +37,22 @@
 /obj/structure/bed/chair/verb/rotate()
 	set name = "Rotate Chair"
 	set category = "Object"
-	set src in oview(1)
+	set src in view(0)
```

Comment:
Not documented in the changelog. 

Also I'd suggest adding a control statement on the buckled mob's a_intent rather than this (that implying your issue is people spinning adjacent occupied chairs), because it's quite wonky having to stand on the same turf to rotate it.

Reply:
Spooky suggested this change, instead of an `Adjacent()` check.
Mostly due to rotating bypassing windows and whatnot, and to avoid that check.

Reply:
Changelog added.

Reply:
Well, fine.

Reply:
I don't mind either way, personally.